spec/lib/guardian_spec.rb

```diff
@@ -95,13 +99,6 @@
       expect(Guardian.new(user).can_send_private_message?(another_user)).to be_truthy
     end
 
-    it "disallows pms to other users if trust level is not met" do
-      SiteSetting.personal_message_enabled_groups = Group::AUTO_GROUPS[:trust_level_2]
-      user.update!(trust_level: TrustLevel[1])
-      Group.user_trust_level_change!(user.id, TrustLevel[1])
-      expect(Guardian.new(user).can_send_private_message?(another_user)).to be_falsey
-    end
```

Comment:
This is a duplicate of https://github.com/discourse/discourse/blob/main/spec/lib/guardian_spec.rb#L87-L92